src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -56,6 +56,8 @@ def to_internal_value(self, data):
         # Update data from cleaned form values
         data.update(form.cleaned_data)
 
+        if getattr(form, "_pending_save", False) and form._pending_save:
```

Comment:
Why not

```py
data["pending_save"] = getattr(form, "_pending_save", False) and form._pending_save
```

is the lack of `pending_save` key in the output important?

Reply:
I don't think the `and form._pending_save` is needed here. `getattr` will already fetch the attribute.